tests/zfs-tests/tests/functional/acl/acl_common.kshlib

```diff
@@ -410,7 +410,7 @@ function get_xattr #<obj>
 	fi
 
 	for xattr in `runat $obj ls | \
-		/usr/xpg4/bin/egrep -v -e SUNWattr_ro -e SUNWattr_rw` ; do
+		/usr/bin/egrep -v -e SUNWattr_ro -e SUNWattr_rw` ; do
```

Comment:
Let's use `grep -E` instead since `egrep` is already technically deprecated.  Yes, I realize this throws away the only part of this OpenZFS change which was applicable for Linux.  It'd be reasonable to fold this change in to #7040 and then comment in the OpenZFS tracker that this change isn't applicable for Linux.

Reply:
I agree with @behlendorf, let's push #7040 instead of egrep.

Reply:
I'm ok with folding this into #7040 if you don't mind refreshing it again.